### DISCUSSION TITLE
Spam, spam, spam and more spam, on Ctrlr.org

### DISCUSSION THREAD

**Discussion by Andy2No** [RomanKubiak/ctrlr#351] [General]
I just followed a link to https://ctrlr.org/behringer-odyssey-patch-saver/ and noticed that a comment there, and all recent topics listed on the right hand margin, are 100% spam.

I wasn't aware it was even possible to post there any more.

**Comment by Godlike-Productions** [comment]
My guess is that it is a wordpress security issue.  Wordpress is a security nightmare.
